src/include/pbs_share.h

```diff
@@ -44,7 +44,9 @@
 
 /* Formula special case constants */
 
-#define FORMULA_FSPERC 		"fair_share_perc"
+#define FORMULA_FSPERC 		"fairshare_perc"
+#define FORMULA_FSPERC_DEP	"fair_share_perc"
```

Comment:
I looked into this.  It turns out that job_sort_key is broken.  You're supposed to be able to use 'fair_share_perc' in it, but you can't.  If you add fair_share_perc into job_sort_key you are told it is invalid.  Since this is broken to begin with, I'd rather not touch it.  You can't put in preempt_prio either.  Not that preempt_prio does anything anymore.  The scheduler always sorts by preemption priority.  I filed PP-819 for fixing or deprecating the job_sort_key keywords